DHS_Recode_V.do

```diff
@@ -19,27 +18,27 @@ macro drop _all
 //NOTE FOR WINDOWS USERS : use "/" instead of "\" in your paths
 
 //global root "C:/Users/wb500886/WBG/Sven Neelsen - World Bank/MEASURE UHC DATA"
-global root "C:\Users\Guan\OneDrive\DHS\MEASURE UHC DATA"
+global root "C:/Users/Vero Comp/Documents/DataWhale/WB/MEASURE UHC DATA"
 
 * Define path for data sources
 global SOURCE "${root}/RAW DATA/Recode V"
 
 * Define path for output data
-global OUT "${root}/STATA/DATA/SC/FINAL_V"
+global OUT "${root}/STATA/DATA/SC/FINAL"
 
 * Define path for INTERMEDIATE
-global INTER "${root}/STATA/DATA/SC/INTER_V"
+global INTER "${root}/STATA/DATA/SC/INTER"
 
 * Define path for do-files
 //global DO "C:\Users\wb500886\OneDrive - WBG\10_Health\UHC\GitHub\DHS-Recode-V"
-global DO "${root}/STATA/DO/SC/DHS/Recode V/Git-DHS-Recode-V"
+global DO "${root}/STATA/DO/SC/Recode_V_training"
 
 
 * Define the country names (in globals) in by Recode
 do "${DO}/0_GLOBAL.do"
 //$DHScountries_Recode_V
 
-foreach name in $DHScountries_Recode_V {	
+foreach Niger2006 in $DHScountries_Recode_V {	
```

Comment:
name is global, which will be referred to several times in the command as `name'. Should not replace name as Niger2006. Instead, when you are trying to work on Niger2006, you should temporarily mute the original command in lin39 (where the global is set using the dofile 0_GLOBAL.do) and set the global DHScountries_Recode_V "Niger2006" temporarily to run your code.

Reply:
Please see the Stata guide here https://wlm.userweb.mwn.de/Stata/wstatmac.htm in terms of how to use global in stata